docs/components/modeler/element-templates/defining-templates.md

```diff
@@ -1,950 +1,104 @@
 ---
 id: defining-templates
 title: Defining templates
-description: "Learn about JSON schema compatibility, supported BPMN types, defining template properties, and more."
+description: "Learn how to define an element template"
 ---
 
-Templates are defined in template descriptor files as a JSON array:
+An element template is defined in template descriptor files as a JSON object.
+The element template object is divided into required and optional keys:
 
-```json
-[
-  {
-    "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
-    "name": "Template 1",
-    "id": "sometemplate",
-    "description": "some description",
-    "keywords": [
-      "search alias",
-      "create action"
-    ],
-    "version": 1,
-    "engines": {
-      "camunda": "^8.6"
-    },
-    "appliesTo": [
-      "bpmn:Task"
-    ],
-    "elementType": {
-      "value": "bpmn:ServiceTask"
-    },
-    "properties": [
-      ...
-    ]
-  },
-  {
-    "name": "Template 2",
-    ...
-  }
-]
-```
-
-As seen in the code snippet, a template consists of a number of important components:
-
-- `$schema : String`: URI pointing towards the [JSON schema](https://json-schema.org/) which defines the structure of the element template `.json` file. Element template schemas are maintained in the [element templates JSON schema](https://github.com/camunda/element-templates-json-schema) repository. Following the [JSON schema](https://json-schema.org/) standard, you may use them for validation or to get assistance (e.g., auto-completion) when working with them in your favorite IDE.
-
-:::note
-
-The `$schema` attribute is **required** for Camunda 8 element templates.
```

Comment:
It is super important that the template creator includes `$schema` as we identify schema-less templates as C7 templates.

Reply:
The key is listed in the ***required*** section. I don't think it's necessary to explicitly mention it again. 
I also don't see a reason, why Camunda 8 should be mentioned explicitly? After all this is the C8 docs and there is dedicated docs for [C7 templates](https://docs.camunda.org/manual/latest/modeler/element-templates/)